Add emulator.wtf sample project

madisp opened the pull request:
Add a sample for running tests with emulator.wtf, copied from the AWS device farm sample.

madisp commented:
Note: I copied over the `FCI_*` env var bits but I'm unsure whether they work the way I expect them to work. I assume this repo is also being used in Codemagic internally for some testing?

VeArnold commented:
If possible, could you replace all the `FCI_` prefixes with `CM_` :)

nevercode-kevin reviewed (changes requested):
Please move the sample to the **integrations** folder.

[codemagic-sample-projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects)/integrations/

madisp commented:
Thanks for the feedback, changes introduced:

- moved to `integrations/emulatorwtf-android-demo-project`
- replaced `FCI_` env var prefix with `CM_`
